doc/effects.md

```diff
@@ -180,14 +180,67 @@ Usage example:
 ```dart
 final combination = CombinedEffect(
     effects: [move, scale, rotate],
-    isInfinite: true, 
+    isInfinite: true,
     isAlternating: true);
 myComponent.addEffect(combination);
 ```
 
 An example of how to use the `CombinedEffect` can be found
 [here](https://github.com/flame-engine/flame/tree/main/examples/lib/stories/effects/combined_effect.dart).
 
+## Common for paint effects
```

Comment:
I think instead of having `## Common for MoveEffect, ScaleEffect and RotateEffect (SimplePositionComponentEffects)` and this, we could have second-level `##` with the type of effect, eg `## Position Effects`, `## Paint Effects`, followed by a brief desc and second level `###` chapters for each type (here and above). Wdyt?

Reply:
Yeah, I think it could look better indeed, but I just followed the current structure. Changing to your proposed structure would make me need to change more things on the doc unrelated to this PR, so I think we could do it, but in a another PR. wdyt?

Reply:
Sure, it can be a followup